src/unleash-proxy.ts

```diff
@@ -144,10 +146,13 @@ export default class UnleashProxy {
                     ...standardResponses(401, 500, 501, 503),
                     200: featuresResponse,
                 },
-                description:
-                    'This endpoint returns the list of feature toggles that the proxy evaluates to enabled and disabled for the given context. As such, this endpoint always returns all feature toggles the proxy retrieves from unleash. Useful if you are migrating to unleash and need to know if the feature flag exists on the unleash server. However, using this endpoint will increase the payload size transmitted to your applications. Context values are provided as query parameters.',
+                description: `This endpoint accepts a JSON object with a \`context\` property and an optional \`toggles\` property.
+
+If you provide the \`toggles\` property, the proxy will use the provided context value to evaluate each of the toggles you sent in. The proxy returns a list with all the toggles you provided in their fully evaluated states.
+
+If you don't provide the \`toggles\` property, then this operation functions exactly the same as the GET operation on this endpoint, except that it uses the \`context\` property instead of query parameters. The proxy will evaluate all its toggles against the context you provide and return a list of all known feature toggles and their evaluated state.`,
```

Comment:
Might be obvious but we can mention that for performance you may want to evaluate the smallest set of toggles possible. 

Also, note you could extract most of this description into a constant to keep it in sync with the other description that resembles this one

Reply:
Hmm, good point. I'll consider it. At the same time, it's only done twice, and I'm thinking it might be incidental that they are the same.